framework/content/ecl-typography/ecl-typography-headings/_headings.scss

```diff
@@ -1,46 +1,75 @@
 // Headers
-.ecl-h1,
-.ecl-h2,
-.ecl-h3,
-.ecl-h4,
-.ecl-h5,
-.ecl-h6 {
+.ecl-heading {
   margin: 0;
-  padding: 0.5em 0;
+  padding: 0;
```

Comment:
I took the wiki documentation as reference, and they speak about line height, but not padding